library/tools/smt2/syntax.lean

```diff
@@ -0,0 +1,85 @@
+namespace smt2
+
+-- not sure about this one
+@[reducible] def symbol : Type := string
+@[reducible] def identifier : Type := string
+
```

Comment:
If you don't mark `symbol` and `identifier` as `[reducible]`, then you will have to define type class instances for them (e.g., `decidable_eq`, `has_to_string`, etc).